src/client_scripts/toggleAudio.ts

```diff
@@ -0,0 +1,17 @@
+import { pc } from './setUpMedia';
+
+export async function toggleAudio (): Promise<void> {
```

Comment:
These two components looks very similar, could we refactor it to use a common base somehow?